.travis.yml

```diff
@@ -25,6 +25,9 @@ env:
         -e UPLOAD_DEST
```

Comment:
From TravisCI.org build log:

    [NOTICE] Replace UPLOAD_DEST environment variable with ENDPOINTS_PACK in .travis.yml

```suggestion
        -e ENDPOINTS_PACK
```

Reply:
Changed.